lib/backend/etcdbk/etcd_test.go

```diff
@@ -120,6 +121,11 @@ func (s *EtcdSuite) TestCompareAndSwap(c *check.C) {
 	s.suite.CompareAndSwap(c)
 }
 
+// Issue: https://github.com/gravitational/teleport/issues/4786
+func (s *EtcdSuite) TestCompareAndSwapIssue4786(c *check.C) {
+	s.suite.CompareAndSwapIssue4786(c)
+}
```

Comment:
I think just having a comment referencing the issue number is fine, we probably don't need to put it in the actual function name.

```suggestion
func (s *EtcdSuite) TestCompareAndSwapOversizedValue(c *check.C) {
	s.suite.TestCompareAndSwapOversizedValue(c)
}
```